### DISCUSSION TITLE
Add Big Search Box to Small Taskbar Icons

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [valinet/ExplorerPatcher#2956] [Ideas]
The big Search Box that was recently added is not in the small taskbar icons feature. (Taskbar icon size feature for small)
Is it possible to add in the future?

**Comment by Amrsatrio** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/df8c24aa-f292-4abc-ae5f-1c9cdd46ce37)

Soon.
